safe/gui/tools/help/multi_buffer_help.py

```diff
@@ -75,9 +75,17 @@ def content():
         'remove, then click the remove (-) button.'
     ))
     tips.add(tr(
+        'To launch keywords creation wizard right after the buffering is '
+        'complete, check the launch keywords wizard check box. If you want '
+        'to do it later, uncheck the launch keywords wizard check box.'
```

Comment:
'Check the "launch keywords wizard" checkbox to launch the 
'keywords creation wizard after the buffering is '
'complete. If you want assign keywords later, uncheck '
'the "launch keywords wizard" checkbox.'